sqlmesh/cli/main.py

```diff
@@ -364,10 +364,27 @@ def dag(ctx: click.Context, file: str) -> None:
 @opt.match_pattern
 @opt.verbose
 @click.argument("tests", nargs=-1)
+@click.option(
+    "--no-truncate",
+    is_flag=True,
+    default=False,
+    help="Show the maximum number of mismatching columns in case of failure.",
+)
 @click.pass_obj
 @error_handler
-def test(obj: Context, k: t.List[str], verbose: bool, tests: t.List[str]) -> None:
+def test(
+    obj: Context,
+    k: t.List[str],
+    verbose: bool,
+    tests: t.List[str],
+    no_truncate: bool,
+) -> None:
     """Run model unit tests."""
+    if no_truncate:
```

Comment:
this needs to be done in the test so it can be reused, you need to add this interface to context, and notebooks as well right?

Reply:
Yeah good point, I'll take a look and see if I can reuse the `-v` flag otherwise I'll address this as well.